pkg/standalone-utils/test/foundry/PauseHelper.t.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { PoolMock } from "@balancer-labs/v3-vault/contracts/test/PoolMock.sol";
+import { PoolFactoryMock } from "@balancer-labs/v3-vault/contracts/test/PoolFactoryMock.sol";
+import { BaseVaultTest } from "@balancer-labs/v3-vault/test/foundry/utils/BaseVaultTest.sol";
+import { IAuthentication } from "@balancer-labs/v3-interfaces/contracts/solidity-utils/helpers/IAuthentication.sol";
+
+import { PauseHelper } from "../../contracts/PauseHelper.sol";
+
+contract PauseHelperTest is BaseVaultTest {
+    PauseHelper pauseHelper;
+
+    function setUp() public virtual override {
+        BaseVaultTest.setUp();
+
+        address[] memory owners = new address[](1);
+        owners[0] = address(this);
+
+        pauseHelper = new PauseHelper(vault);
+
+        authorizer.grantRole(pauseHelper.getActionId(pauseHelper.addPools.selector), address(this));
+        authorizer.grantRole(pauseHelper.getActionId(pauseHelper.removePools.selector), address(this));
+        authorizer.grantRole(pauseHelper.getActionId(pauseHelper.pause.selector), address(this));
+
+        authorizer.grantRole(vault.getActionId(vault.pausePool.selector), address(pauseHelper));
+    }
+
+    function testAddPoolsWithTwoBatches() public {
+        address[] memory firstPools = _addPools(10);
+        assertEq(pauseHelper.getPoolsCount(), firstPools.length, "Pools count should be 10");
+
+        address[] memory secondPools = _addPools(10);
+        assertEq(pauseHelper.getPoolsCount(), firstPools.length + secondPools.length, "Pools count should be 20");
+
+        for (uint256 i = 0; i < firstPools.length; i++) {
+            assertTrue(pauseHelper.hasPool(firstPools[i]));
+        }
+
+        for (uint256 i = 0; i < secondPools.length; i++) {
+            assertTrue(pauseHelper.hasPool(secondPools[i]));
+        }
```

Comment:
Could check that an invalid address returns false:
`assertFalse(pauseHelper.hasPool(address(pauseHelper), "Has invalid pool");`
`assertFalse(pauseHelper.hasPool(address(0), "Has zero address pool");`